src/components/notebook/macro-panel.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "@testing-library/jest-dom";
```

Comment:
You shouldn't need this since it's already in `setupTests.ts`.

Reply:
Fixed.